### ISSUE TITLE
MP bar not filling on a server

### ISSUE THREAD

**Issue by GoldenMaster8** [MoriyaShiine/bewitchment#136]
I run a server with bewitchment 1.16.5.14 (aka the newest version) along wih a few other mods, and when anyone joins the server the mana bar just remains on their screen, and even when near an altar, or eating a witchberry cookie/pie, the bar doesnt fill up. I tried to test it in singleplayer, even on the same world, and it works just fine.

Mod list is in PNG 
<img width="1301" alt="Screen Shot 2021-05-19 at 8 20 47 AM" src="https://user-images.githubusercontent.com/78369802/118811562-24a1be00-b87b-11eb-9d1c-452b2a114297.png">

**Comment by UwUE11ie** [comment]
dude if you have a public server could I please join, I've been looking for a bewitchment server FOREVER.

**Comment by MoriyaShiine** [comment]
Can you see if this happens with just Bewitchment? Sounds like a desync

**Comment by HyperlyBola** [comment]
I'm having an issue where the bar isn't disappearing a few seconds after joining

**Comment by MoriyaShiine** [comment]
All servers have logs, even vanilla. Please post them here. Don't change the texture because this indicates a desync and will be impossible to know when it happens if the texture is invisible. Solving a problem is better than ignoring it

**Comment by HyperlyBola** [comment]
I decided to check logs using my multicraft server instead of the default vanilla console. I also noticed some weird logs on the client game output

[https://pastebin.com/4HkvpQSA](url)
[https://pastebin.com/GfaEWBhU](url) (two different logs)

Will just say that making it invisible is temporary, and by now after testing a list of 70 mods and carefully picking out each I'm fairly certain I know when it happens. I don't fully understand the logs so I don't know how much it can help. I guess it would help to list out conditions so here..

How to reproduce:
- Have bewitchment, besmirchment, and the normal library installed
- (optionally have phosphor and lithium installed)
- Include one of those mods from the list I gave in your mods folder (preferably Haema)
- Run a server with default settings
- Join server
